math/big/decimal.go

```diff
@@ -2283,6 +2283,13 @@ func NewDecFromFloatForTest(f float64) *Decimal {
 	return dec
 }
 
+// NewDecFromFloat creates a Decimal from float.
+func NewDecFromFloat(f float64) (*Decimal, error) {
+	dec := &Decimal{}
+	err := dec.FromFloat64(f)
+	return dec, err
+}
+
 // NewDecFromStringForTest creates a Decimal from string, as it returns no error, it should only be used in test.
 func NewDecFromStringForTest(s string) *Decimal {
```

Comment:
now we can use  NewDecFromString for ut test, the method can remove now ?

Reply:
i'll remove it